users/views.py

```diff
@@ -11,3 +16,29 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data)
+
+
+class LoginView(APIView):
+    def post(self, request):
+        username = request.data["username"]
+        password = request.data["password"]
+
+        user = User.objects.filter(username=username).first()
+
+        if user is None:
+            raise AuthenticationFailed("404 - User does not exist")
+
+        if not user.check_password(password):
+            raise AuthenticationFailed("401 - Incorrect password")
+
+        payload = {
+            "username": user.username,
```

Comment:
lets add issued at or "iat" key with UNIX epoch time. I am not sure what it will be used for, but we will figure out